firefox-code-search/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I think this isn't shipping until Firefox 52? So it would be good to have an `applications` key with `strict_min_version` of 52.0a1 (or something).